README.md

```diff
@@ -18,9 +18,72 @@ Or install it yourself as:
 
     $ gem install grpc-kit
 
-## Usage
+## Documentation
 
-TODO: Write usage instructions here
+- [`GRPC::Kit::Communication::Resilient`](#grpckitcommunicationresilient)
+- [`GRPC::Kit::Logger`](#grpckitlogger)
+- [`GRPC::Kit::Queue::Publisher`](#grpckitqueuepublisher)
+- [`GRPC::Kit::Queue::Worker`](#grpckitqueueworker)
+
+### GRPC::Kit::Communication::Resilient
+
+Sometimes, GRPC and Google::Cloud raises _unavailable error_, but you can retry and all works fine. Based on [Datastore documentation](https://cloud.google.com/appengine/articles/handling_datastore_errors#timeouts-due-to-datastore-issues) we implemented a _helper_ to _resilient communication_.
```

Comment:
acho que ficaria melhor `for`  no lugar de `to` aqui em _resilient communication_